test/test_helper.rb

```diff
@@ -0,0 +1,11 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/node'
+require './lib/binary_search_tree'
+
```

Comment:
for the test helper to work everywhere, you need to give it _every_ require you'll need in your project.

Obviously you can add something like `pry` here, and it'll be available in all of your tests.